includes/themes.php

```diff
@@ -964,6 +966,26 @@ public function install_theme( $file ) {
 	public function sync_settings() {
 		_deprecated_function( __METHOD__, '5.0' );
 	}
+
+	/**
+	 * Check current theme requires plugin version and show admin notice if it doesn't meet the requirement  .
+	 *
+	 * @returns {void}
+	 */
+	private function check_theme_requiered_version() {
```

Comment:
```suggestion
	private function check_theme_required_version() {
```
Can we update the typo here?